Source/WebCore/en.lproj/Localizable.strings

```diff
@@ -901,6 +901,12 @@
 /* Pause All Animations context menu item */
 "Pause All Animations" = "Pause All Animations";
 
+/* Show Subtitles context menu item */
+"Show Subtitles" = "Show Subtitles";
+
+/* Hide Subtitles context menu item */
+"Hide Subtitles" = "Hide Subtitles";
```

Comment:
Instead of inserting these by hand you can run `update-webkit-localizable-strings` (one benefit is that the script maintain the strings in sorted order).

Reply:
Thanks!